SlicerCART/src/SlicerCART.py

```diff
@@ -1266,31 +1281,52 @@ def __init__(self, segmenter, modality, edit_conf, label_config_yaml = None, par
                     cell.setForeground(qt.QBrush(qt.QColor(0, 0, 0)))
                     self.label_table_view.setItem(index, 6, cell)
                     self.label_table_view.setHorizontalHeaderItem(6, qt.QTableWidgetItem('Max. HU'))
+                    
+      self.label_table_view.setSizePolicy(qt.QSizePolicy.Expanding, qt.QSizePolicy.Fixed)
 
       self.add_label_button = qt.QPushButton('Add Label')
       self.add_label_button.clicked.connect(self.push_add_label)
       layout.addWidget(self.add_label_button)
 
-      self.save_button = qt.QPushButton('Save')
-      self.save_button.clicked.connect(self.push_save)
-      layout.addWidget(self.save_button)
+      display_timer_hbox = qt.QHBoxLayout()
+
+      self.display_timer_label = qt.QLabel('Display timer during segmentation? ')
+      self.display_timer_label.setStyleSheet("font-weight: bold")
+      display_timer_hbox.addWidget(self.display_timer_label)
+
+      self.display_timer_checkbox = qt.QCheckBox()
+      display_timer_hbox.addWidget(self.display_timer_checkbox)
+
+      layout.addLayout(display_timer_hbox)
+
+      self.apply_button = qt.QPushButton('Apply')
+      self.apply_button.clicked.connect(self.push_apply)
+      layout.addWidget(self.apply_button)
 
       self.cancel_button = qt.QPushButton('Cancel')
       self.cancel_button.clicked.connect(self.push_cancel)
-      layout.addWidget(self.cancel_button)
+      layout.addWidget(self.cancel_button)      
+
+      self.populate_default_values()
```

Comment:
This function populate_default_values() was removed previously in the script, but was found in the original script. Is it expected? Now, this may cause errors since the function is not defined and is only called.

Reply:
Found later in the script.

Reply:
Reason to have it moved?